Sources/Datadog/RUM/RUMEvent/RUMEventEncoder.swift

```diff
@@ -69,3 +69,5 @@ extension RUMActionEvent: RUMSanitizableEvent {}
 extension RUMResourceEvent: RUMSanitizableEvent {}
 
 extension RUMErrorEvent: RUMSanitizableEvent {}
+
+extension RUMLongTaskEvent: RUMSanitizableEvent {}
```

Comment:
This needs an update in `RUMEventSanitizerTests` where we cover all sanitizable events.